src/consensus/pbft_manager.cpp

```diff
@@ -1586,7 +1586,8 @@ void PbftManager::finalize_(PbftBlock const &pbft_block, vector<h256> finalized_
 
 bool PbftManager::pushPbftBlock_(PbftBlockCert const &pbft_block_cert_votes, vec_blk_t const &dag_blocks_order,
                                  bool sync) {
-  auto const &pbft_block_hash = pbft_block_cert_votes.pbft_blk->getBlockHash();
+  auto pbft_block = pbft_block_cert_votes.pbft_blk;
```

Comment:
you could use just hash here and even in `getJsonStrForBlock` as you are not using anything except that

Reply:
Don't getting what about is in the first part of your message, because pbft_block using below in some lines. And I just moved that line above
And good point about `getJsonStrForBlock`, changed that

Reply:
sorry my bad I did not notice